### ISSUE TITLE
dipy.data: three_shells_voxels is not there

### ISSUE THREAD

**Issue by arokem** [nipy/dipy#560]
```
In [1]: import dipy.data as dpd

In [2]: data = dpd.three_shells_voxels(10, 20, 10, 20, 10, 20)
---------------------------------------------------------------------------
IOError                                   Traceback (most recent call last)
<ipython-input-2-3eb0f169ff0c> in <module>()
----> 1 data = dpd.three_shells_voxels(10, 20, 10, 20, 10, 20)

/Users/arokem/anaconda3/envs/py2/lib/python2.7/site-packages/dipy/data/__init__.pyc in three_shells_voxels(xmin, xmax, ymin, ymax, zmin, zmax)
    364 def three_shells_voxels(xmin,xmax,ymin,ymax,zmin,zmax):
    365     fimg, fbvals, fbvecs = get_data('3shells_data')
--> 366     bvals = np.loadtxt(fbvals)
    367     bvecs = np.loadtxt(fbvecs).T
    368     bvecs[:,0] = -bvecs[:,0]

/Users/arokem/anaconda3/envs/py2/lib/python2.7/site-packages/numpy/lib/npyio.pyc in loadtxt(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin)
    735                 fh = iter(bz2.BZ2File(fname))
    736             elif sys.version_info[0] == 2:
--> 737                 fh = iter(open(fname, 'U'))
    738             else:
    739                 fh = iter(open(fname))

IOError: [Errno 2] No such file or directory: '/Users/arokem/anaconda3/envs/py2/lib/python2.7/site-packages/dipy/data/3shells-1000-2000-3500-N193.bval'
```

**Comment by arokem** [comment]
Also: 

```

In [6]: dpd.two_shells_voxels(10, 20, 10, 20, 10, 20)
---------------------------------------------------------------------------
IOError                                   Traceback (most recent call last)
<ipython-input-6-5f354229870b> in <module>()
----> 1 dpd.two_shells_voxels(10, 20, 10, 20, 10, 20)

/Users/arokem/anaconda3/envs/py2/lib/python2.7/site-packages/dipy/data/__init__.pyc in two_shells_voxels(xmin, xmax, ymin, ymax, zmin, zmax)
    351 def two_shells_voxels(xmin,xmax,ymin,ymax,zmin,zmax):
    352     fimg, fbvals, fbvecs = get_data('ISBI_testing_2shells_table')
--> 353     bvals = np.loadtxt(fbvals)
    354     bvecs = np.loadtxt(fbvecs).T
    355     gtab = gradient_table(bvals[1:], bvecs[1:,:])

/Users/arokem/anaconda3/envs/py2/lib/python2.7/site-packages/numpy/lib/npyio.pyc in loadtxt(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin)
    735                 fh = iter(bz2.BZ2File(fname))
    736             elif sys.version_info[0] == 2:
--> 737                 fh = iter(open(fname, 'U'))
    738             else:
    739                 fh = iter(open(fname))

IOError: [Errno 2] No such file or directory: '/Users/arokem/anaconda3/envs/py2/lib/python2.7/site-packages/dipy/data/2shells-1500-2500-N64.bval'

```

**Comment by samuelstjean** [comment]
It's hosted on someone dropbox, maybe @Garyfallidis, isn't there some space available on github or someone webserver for that?

**Comment by arokem** [comment]
OK - seems like this might be cruft. We should remove it. Will make a PR.

**Comment by samuelstjean** [comment]
Strangely enough trying to import it doesn't trigger the fetcher, but the dataset exist through the proper function, 

In [13]: from dipy.data import fetch_sherbrooke_3shell, read_sherbrooke_3shell

In [14]: fetch_sherbrooke_3shell()
Dataset is already in place. If you want to fetch it again, please first remove the folder /home/samuel/.dipy/sherbrooke_3shell

So maybe thta part of code should indeed be reviewed and investigated. Isn't there something to these these fetcher/import dataset function?

**Comment by arokem** [comment]
Just for the record: @samuelstjean - I think that you are referring to a different data-set than the one mentioned in this issue and associated PR.

**Comment by stefanv** [comment]
Closed by #563
